### PR TITLE
Make whisper retention settings configurable

### PR DESCRIPTION
**Pull request by zmc**
Fixes https://github.com/ceph/cephmetrics/issues/61

### CODE REVIEW

File: ansible/roles/ceph-grafana/tasks/configure_carbon.yml [hunk 1]
```diff
@@ -8,10 +8,15 @@
   notify:
     - Restart carbon-cache
 
+- name: Fail when the the first whisper retention value is not 10s
+  fail:
+    msg: "The first whisper retention value must be '10s', not '{{ whisper.retention[0][0] }}'"
```
Comment (GregMeno) [inline ansible/roles/ceph-grafana/tasks/configure_carbon.yml]: It's great that we have a check to make sure carbon is reporting at the same interval as our most precise retention.
It'd be better still if we said that correlation was the reason for the check.

File: ansible/roles/ceph-grafana/defaults/main.yml [hunk 2]
```diff
@@ -21,6 +21,11 @@ carbon:
   storage_dir:
     yum: /var/lib/carbon
     apt: /var/lib/graphite
+whisper:
+  retention:
+    - ['10s', '7d']
```
Comment (GregMeno) [inline ansible/roles/ceph-grafana/defaults/main.yml]: A comment describing the format of this data would go a long way e.g. # 10s interval retention for the first 7 days worth of data. What do you think?

Comment (pcuzner) [inline ansible/roles/ceph-grafana/defaults/main.yml]: Agree. I think the PR should probably include an update to the README so an admin who's installing  picks up on the option to change retention and the implications.

Comment (zmc) [inline ansible/roles/ceph-grafana/defaults/main.yml]: Pushed a commit with a README entry

**Pull request merged**